efficientdet/keras/efficientdet_keras.py

```diff
@@ -932,12 +955,22 @@ def _postprocess(self, cls_outputs, box_outputs, scales, mode='global'):
     if not mode:
       return cls_outputs, box_outputs
 
+    # TODO(tanmingxing): remove this cast once FP16 works postprocessing.
+    cls_outputs = [tf.cast(i, tf.float32) for i in cls_outputs]
```

Comment:
It could be removed since casts are done in to_list.

Reply:
I feel the type conversion should happen here, instead of to_list.
I didn't into much detail for this part, but what's the reason to remove these casts, and do the cast in to_list?